assets/js/base/components/product-list/index.js

```diff
@@ -9,13 +9,16 @@ import ProductListItem from '@woocommerce/base-components/product-list-item';
 import {
 	useStoreProducts,
 	useSynchronizedQueryState,
+	useQueryStateByKey,
 } from '@woocommerce/base-hooks';
 import withScrollToTop from '@woocommerce/base-hocs/with-scroll-to-top';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
+import noProducts from './no-products';
+import noMatchingProducts from './no-matching-products';
```

Comment:
Just curious here, why making these two a function instead of a React component (using a capital letter, calling them as `<NoMatchingProducts resetCallback={ ... } />`, etc.)?

Reply:
Because I have no idea what I'm doing.